### PR TITLE
New Milestone : JavaScript Pop Up Feature

### PR DESCRIPTION
**Pull request by toyman640**
I added a pop up section to the html files.
I created JavaScript Object which provides dynamic information to the pop up card.
Dynamic properties in the pop up  card includes Title, Job history and card images.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -258,12 +307,71 @@ <h3 class="title-1">
...
+            </button>
+            <button class="desk-see-proj desk-pop-button">
+              <a class="desk-project-link desk-pop-link" href="">See source <img src="images/see-source-logo.svg" alt=""></a>
+            </button>
+          </div>
+        </div>
+      </div>
+    </div>
+    <section class="desk-works flex-column for-desktop">
       <div class="works-card">
```
Comment (Whoistolu) [inline index.html]: - [ ] [REQUIRED] Let's ensure that the desktop popup follows the design [template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, images, text, margins). You need to work on the [X] button, it needs to be well-positioned closer to the edge of the div. Could you check the template provided?
- [ ] Also, the desktop popup is way too bigger for the screen when it pops. Kindly reduce the width a bit so it fits well on the screen when the `see project` button is been clicked.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
Screenshot 2023-04-12 at 11 16 55 | Screenshot 2023-04-12 at 11 31 46

File: index.html [hunk 2]
```diff
@@ -83,6 +83,54 @@ <h4 class="connect">LET’S CONNECT</h4>
         </div>
       </div>
     </section>
+    <div class="item works-1 pop invisible-pop" id="work-1">
+      <div class="card-content pop-main">
+        <img class="pop-close" src="images/pop-close.svg" alt="">
```
Comment (Whoistolu) [inline index.html:88]: - [ ] [REQUIRED] Great job with the mobile popup 👍🏽  However, we have another close button [X] beneath the modal right on top of the `See live` and `See source` buttons that should be removed. Kindly remove this as it is not in the design template provided. 

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
Screenshot 2023-04-12 at 11 15 59 | Screenshot 2023-04-12 at 11 42 57

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @toyman640 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

- [ ] [OPTIONAL] Kindly remove the Live demo part of the README since this project is yet to be deployed and it is not a requirement for this milestone.

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by binhussen — APPROVED** [review]
Hi @toyman640 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**